DependencyInjection/Compiler/StaticAsseticHelperPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Symfony\Bundle\AsseticBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class StaticAsseticHelperPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('assetic.helper.static')) {
+            return;
+        }
+
+        $definition = $container->getDefinition('assetic.helper.static');
+
+        if (!method_exists($definition, 'setShared')) {
+            $definition->setScope('request');
```

Comment:
well, because we don't always need our helper to be request-scoped even in 2.7 and older, and it is better to avoid scoping it when not needed.
See https://github.com/symfony/symfony/blob/2.6/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/TemplatingAssetHelperPass.php for the logic determining the scope of the asset helper conditionally